fdbrpc/include/fdbrpc/DDSketch.h

```diff
@@ -205,7 +208,7 @@ class DDSketchBase {
 	double errorGuarantee; // As defined in the paper
 
 	uint64_t populationSize, zeroPopulationSize; // we need to separately count 0s
-	std::vector<uint64_t> buckets;
+	std::vector<uint32_t> buckets;
```

Comment:
Changing to `uint32_t` to save space. With the default error guarantee, it is highly unlikely a given bucket could overflow. This allows a higher precision with the same amount of memory as before.